eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/zmq/ZMQActor.scala

```diff
@@ -18,7 +18,7 @@ package fr.acinq.eclair.blockchain.bitcoind.zmq
 
 import akka.Done
 import akka.actor.{Actor, ActorLogging}
-import fr.acinq.bitcoin.scalacompat.{ByteVector32, Transaction}
+import fr.acinq.bitcoin.scalacompat.{BlockHash, BlockId, ByteVector32, Transaction}
```

Comment:
Unused header.

Reply:
Thanks, removed in https://github.com/ACINQ/eclair/pull/2742/commits/23d44b150c8fa27ef5539dabf5688fb6aeb13aff